webpack.config.babel.js

```diff
@@ -211,20 +211,12 @@ config.plugins.push(
   new webpack.DefinePlugin({
     'process.env': {
       BPK_BUILT_AT,
+      GOOGLE_MAPS_API_KEY: JSON.stringify(GOOGLE_MAPS_API_KEY),
```

Comment:
What happens if this is blank will it be ok?

Reply:
It will be 👌